using-datapower-in-kubernetes/getting-started-datapower-in-gke.md

```diff
@@ -0,0 +1,271 @@
+```bash
+$ gcloud config set compute/zone <zone>
+```
+To see valid choices for `<zone>`, run:
+```bash
+$ gcloud compute zones list
+```
+
+### Get the Source Code
+
+```bash
+$ git clone https://github.com/ibm-datapower/datapower-tutorials.git
+$ cd datapower-tutorials/using-datapower-in-kubernetes/
+```
+
+## Tools we will use
+* Docker
+    * We use Docker to package, distribute and run 
+    * https://www.docker.com
+
+* IBM DataPower Geteway for Docker
+    * Available as a Docker image in DockerHub
+    * https://hub.docker.com/r/ibmcom/datapower/
+    * https://developer.ibm.com/datapower/
+
+* Kubernetes
+    * Handle the heavy lifting of orchestrating our application
+    * http://kubernetes.io
+
+* Google Container Engine (GKE)
+    * A hosted Kubernetes service
+    * https://cloud.google.com/container-engine
+
+* Google Cloud Shell
+    * A shell environment for managing Google Cloud Platform resources
+    * https://cloud.google.com/shell/docs/
+
+
+
+#  Introduction to Kubernetes
+
+## Setting Up a Kubernetes cluster with GKE
+
+GKE provides a hosted Kubernetes service that can be customized. We will leverage Kubernetes to handle the complexity 
+associated with orchestrating application containers.
+
+To provision a GKE cluser, open up your project in Google Cloud Platform and start an instance of the Google Cloud Shell.
+Next, run the following command to create a GKE cluster of 3 nodes for use in this tutorial:
+
+```bash
+$ gcloud container clusters create k8s-cluster
+```
+
+This might take a couple of minutes.
+
+With our cluster up and running, we are now ready to run containers. 
+
+
+## Introduction to Pods
+
+In Kubernetes, all containers run in what's called a pod. A pod represents a logical application composed of co-located 
+and co-scheduled containers, shared storage, and certain shared namespaces. Pods, not containers, are the smallest deployable artifact in Kubernetes.
+
+To run a pod directly, we can, for example, run a DataPower container with:
+
+```bash
+$ kubectl run --stdin --tty datapower --image=ibmcom/datapower:latest --env="DATAPOWER_ACCEPT_LICENSE=true" --env="DATAPOWER_INTERACTIVE=true"
+
+```
+
+This will automatically create a default Kubernetes _deployment_ , which is a Kubernetes construct that lets you declare the desired state of
+your application and is used to update your pods or replica sets to try to match the current state of your application to that of the desired state.
+You can specify deployments, pods, services, replica sets, etc. declaratively in Kubernetes by writing a
+corresponding configuration file.
+
+In a separate Google Cloud Shell session, we can run `$ kubectl describe deployments` to show the details
+ of the *datapower* deployment we just created.
+ 
+ ![kubectl describe deployments](media/kubectl_describe_deployments_datapower.png)
+
+Don't worry if not everything from the output makes sense at this point, the key takeaway here is that we can use the `describe` command to 
```

Comment:
"key takeaway" is jargony. Try something like "...just know that you can use the `describe command to..."